backend/src/apiserver/server/run_server.go

```diff
@@ -454,20 +454,6 @@ func (s *RunServer) ReportRunMetricsV1(ctx context.Context, request *apiv1beta1.
 	}, nil
 }
 
-// Fetches run metrics for a given run id.
-// Supports v1beta1 behavior.
-func (s *RunServer) getRunMetrics(ctx context.Context, runId string) ([]*model.RunMetric, error) {
```

Comment:
Actually, when did this become dead code? I thought we do used to have v1 feature using this, maybe we disconnected the API and the implementation from the last API refactoring?